torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -14,6 +14,25 @@
 namespace torch {
 namespace jit {
 
+c10::optional<Stack> tryConstantPropNode(const Node* node) {
```

Comment:
This is missing all of constant props checks that the node itself can be constant-propagated. Doesn't seem safe to use.

Reply:
It seems like this method is actually use as `runNodeIfInputsAreConstant` which has a different implicit contract. Should document what this will do. What is `Print(4)` expected to do here?